src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -51,4 +52,19 @@
   &__submit-button-icon {
     color: $white;
   }
+
+  &__emoji-picker-toggler {
+    cursor: pointer;
+  }
+
+  &__emoji-picker {
+    position: absolute;
+    bottom: 65px;
+    right: 60px;
+    z-index: 2;
+
+    input {
+      color: $light-grey;
+    }
```

Comment:
I assume we're doing this to override an element within `emoji-mart`?

Instead of this, can we just set one of the props on it instead?:

- https://github.com/missive/emoji-mart#picker

Skimming that list, `color`/`theme` look promising

Reply:
implemented 'dark' theme and remove css for input

Reply:
Was removed